examples/calling-apis/langchain-examples/src/agents/tools/list_repositories.py

```diff
@@ -0,0 +1,33 @@
+from github import Github
+from github.GithubException import BadCredentialsException
+from src.auth0.auth0_ai import with_github_access
+from pydantic import BaseModel
+from langchain_core.tools import StructuredTool
+from auth0_ai_langchain.federated_connections import FederatedConnectionError, get_credentials_for_connection
+
+
+class ListRepositoriesSchema(BaseModel):
+    pass
+
+
+def list_repositories_tool_function():
+    credentials = get_credentials_for_connection()
+
```

Comment:
```python
if not credentials:
        raise ValueError(
            "Authorization required to access the Federated Connection API")
```

Reply:
is it value error? or FederatedConnectionError?

Reply:
check other examples